dev/plugins/hu.elte.txtuml.api.model/src/hu/elte/txtuml/api/model/ModelClass.java

```diff
@@ -45,16 +45,17 @@
  * <li><i>Be abstract:</i> disallowed</li>
  * <li><i>Generic parameters:</i> disallowed</li>
  * <li><i>Constructors:</i> allowed, only with parameters of types which are
- * subclasses of <code>ModelClass</code> or primitives (including {@code String}
- * )</li>
+ * subclasses of <code>ModelClass</code>, data types, signals or primitives
+ * (including {@code String} )</li>
  * <li><i>Initialization blocks:</i> allowed, containing only simple assignments
  * to set the default values of its fields</li>
  * <li><i>Fields:</i> allowed, only of types which are subclasses of
```

Comment:
`ModelClass` attributes are not allowed. Missing enums and data types. What about [externals](https://github.com/ELTE-Soft/txtUML/pull/485/files#r105639193)?